horovod/spark/lightning/remote.py

```diff
@@ -312,7 +312,7 @@ def calculate_shuffle_buffer_size():
         """
         import horovod.torch as hvd
 
-        if user_shuffle_buffer_size:
+        if user_shuffle_buffer_size != -1:
```

Comment:
We should also return 0 if user_shuffle_buffer_size is zero or do zero-check in the caller.

Reply:
what if `user_shuffle_buffer_size < 0` here?

Reply:
I think adding necessary zero-check in caller is more clear, also addresses valid value issue for < 0 case.
Will fix.

Reply:
@Tixxx @EnricoMi fixed.